### PR TITLE
some doc fixes around configuration

### PR DESCRIPTION
**Pull request by tmbo**
**Proposed changes**:
- some fixes around configuration and docs

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog

### CODE REVIEW

File: docs/pipeline.rst [hunk 1]
```diff
@@ -32,24 +32,131 @@ Pre-configured Pipelines
...
+Here is a list of the **existing templates**:
+
+spacy_sklearn
+~~~~~~~~~~~~~
+
+to use it as a template:
+
+.. code-block:: yaml
+
+    language: "en"
```
Comment (Ghostvv) [inline docs/pipeline.rst]: do we want to say smith like `your_language_here`? And also to specify that spacy supports limited number of languages

File: docs/pipeline.rst [hunk 2]
```diff
@@ -32,24 +32,131 @@ Pre-configured Pipelines
...
+
+
+tensorflow_embedding
+~~~~~~~~~~~~~~~~~~~
+
+to use it as a template:
+
+.. code-block:: yaml
+
+    language: "en"
```
Comment (Ghostvv) [inline docs/pipeline.rst]: adding a note here, that theoretically it supports any language that can be tokenized

File: docs/pipeline.rst (hunk 1 above)
Comment (tmbo) [reply docs/pipeline.rst]: I'll keep `en`, I think that is clear, but I've added a link to the languages section :+1:

**Comment by tmbo** [comment]
@Ghostvv mind approving?

**Pull request merged**